ejs-views/pages/error.ejs

```diff
@@ -0,0 +1,16 @@
+<%- include('../partials/header', { page }); %>
+
+<article>
+  <div class="w-full border border-secondary rounded p-3">
+    <span class="text-3xl text-primary">This Page encountered an Error!</span>
```

Comment:
Do we maybe want to add something like "try again in x mins"? Or something else to explain that this is transient or temporary. I know we want to solve the issue in the long run but would this help in the short?

Reply:
Thanks for the review!

For this one, we certainly could add text like that. Although this error will appear now for **any** error on the frontend. So it won't always be true that the issue is transient. eg. This has become the new 404 for packages, and anything else.

What do you think? Does it still make sense? Since sometimes that will be accurate

Reply:
Oh I see, maybe we could explain that it could be a generic error but also show that we do have an ongoing error and if you see it could you report it at github issue/discord? Or would that no longer be useful?

Reply:
Yeah that's not a bad idea, I'll add it in

Reply:
Added a message like we talked about. @Daeraxa Let me know what you think